src/main/java/org/cbioportal/application/file/export/services/CancerStudyMetadataService.java

```diff
@@ -0,0 +1,23 @@
+package org.cbioportal.application.file.export.services;
+
+import java.util.List;
+import org.cbioportal.application.file.export.mappers.CancerStudyMetadataMapper;
+import org.cbioportal.application.file.model.CancerStudyMetadata;
+import org.cbioportal.application.file.model.CancerType;
+
+public class CancerStudyMetadataService {
```

Comment:
This feature has created its own architecture... I would prefer it get pulled out sooner rather than later

We have skipped the Repository layer.

Service -> Mapper 

Our convention has been to wrap these in a Repository Layer

Reply:
@haynescd  Yes, I realized that. Do you see any parts of the code in this feature that could fit well in the Repository layer? Or would it just end up calling the mappers? If that's the case, it doesn’t seem very practical at the moment. I'd prefer to postpone adding the Repository layer until there's a clearer need for it.

Reply:
@haynescd I've added repositories here https://github.com/cBioPortal/cbioportal/pull/11482/commits/b346ac5bb9d169fba3ec250da24d03b3e3ce2442

Please review if that what you meant.